packages/test/test-service-load/src/azureapplicationinsights.ts

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as appinsights from "applicationinsights";
+
+import { ITelemetryBaseEvent } from "@fluidframework/common-definitions";
+import { TelemetryLogger } from "@fluidframework/telemetry-utils";
+import { ITelemetryBufferedLogger } from "@fluidframework/test-driver-definitions";
+
+/**
+ * Logger to send events to Azure Application Insights.
+ * Can be used by defining env var as: FLUID_TEST_LOGGER_PKG_PATH='./azureapplicationinsights'
+ * Define instrumentation key in env var as: APPINSIGHTS_INSTRUMENTATIONKEY='<instrumentation key>'.
+ */
+export class AppInsightsLogger extends TelemetryLogger implements ITelemetryBufferedLogger {
```

Comment:
I don't think you are getting anything from extending `TelemetryLogger`, probably better not to.